README.md

```diff
@@ -165,4 +166,13 @@ fstr.Pln("{NoSuchField}", struct{}{})  // Output: <invalid field>
 - `P(format string, args ...interface{}) (int, error)` - Shorthand for Printf
 - `Pln(format string, args ...interface{}) (int, error)` - Shorthand for Println
 - `Fprintf(w io.Writer, format string, args ...interface{}) (int, error)` - Prints to io.Writer
-- `Fprintln(w io.Writer, format string, args ...interface{}) (int, error)` - Prints to io.Writer with newline
\ No newline at end of file
+- `Fprintln(w io.Writer, format string, args ...interface{}) (int, error)` - Prints to io.Writer with newline
+
+## Benchmarks
+
+To run the benchmarks:
+
+```bash
+go test -bench=. -benchmem
+```
+
```

Comment:
Could you provide results here?

Having to fetch the repository to see if the lib is acceptable or not would be appreciated